### ISSUE TITLE
How can I find the power consumption of connections (links) in iFogsim

### ISSUE THREAD

**Issue by Mohamedabdkhaleq** [Cloudslab/iFogSim1#23]
Hello 
I'm working on reducing power consumption in fog computing.
Can I find the energy consumed in the connections between the nodes (the energy consumed in the links) in the iFogsim or do I have to write a function for this?

**Comment by saeedehbaneshi** [comment]
Hello Mohamed,

I have the same problem did you find the answer?

**Comment by Mohamedabdkhaleq** [comment]
No I'm so sorry

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by saeedehbaneshi** [comment]
So what did you do for power consumption of links? Do you want to model it
yourself?

**Comment by Mohamedabdkhaleq** [comment]
Yes, I will add a function to calculate the energy consumption in the links

احصل على Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Yamna2023** [comment]
hello there did you solve it ???
if yes can you help me

**Comment by Mohamedabdkhaleq** [comment]
The problem was solved by adding a function inside iFogsim to calculate the power consumption of the links

________________________________

**Comment by Yamna2023** [comment]
can you please  share with me  the code source of this function  please 
yam.yamna48@gmail.com

**Comment by saeedehbaneshi** [comment]
Hi,
Could you also please send it to me?
***@***.***
thanks
